src/main/java/Chatbot/alain/Deadlines.java

```diff
@@ -0,0 +1,33 @@
+package chatbot.alain;
+
+/**
+ * Represents a task with a specific deadline.
+ */
+public class Deadlines extends Task {
```

Comment:
Since this class refers to a singular Deadline Object, I feel that the name should be changed to Deadline rather than Deadlines to indicate singularity